src/api/routers/participants/participantsRouter.ts

```diff
@@ -57,21 +55,17 @@ import {
   updateParticipant,
   updateParticipantAndTypesAndApiRoles,
   UpdateSharingTypes,
-  UserParticipantRequest,
+  UserParticipantRequest
```

Comment:
I'm not sure if we have different format settings, but you've got a lot of changes with the extra comma for the last import from a class.  When I go into my portal code and remove the comma and press save, it adds the comma back.  I don't think the format really matters, but we should probably all be on the same page with it so we don't end up going back and forth with comma edits in PRs

Reply:
good catch! turns out my ALT+SHIFT+O command  to remove unused imports also removed the trailing commas. Saving the file added them back